spec/dummy/app/controllers/components_controller.rb

```diff
@@ -2,4 +2,17 @@ class ComponentsController < ApplicationController
   def show
     render params[:name]
   end
+
+  def new_show
+    component_class = "#{params.require(:name)}_cell".camelize.constantize
+    @example = component_class.examples[example_index]
+    fail "Example #{example_index} not found" unless @example
+    render layout: 'single_component'
+  end
```

Comment:
So would this eventually replace the existing `show` method, or would we keep both?